pkg/assets/assets.go

```diff
@@ -16,35 +17,41 @@ type assets struct {
 	config *config.Config
 }
 
-// Data simply provides ldap server data file.
-func (a assets) Data() ([]byte, error) {
-	if a.config.LDAP.Data != "" {
-		if _, err := os.Stat(a.config.LDAP.Data); os.IsNotExist(err) {
-			a.logger.Error().
-				Str("path", a.config.LDAP.Data).
-				Msg("Data file doesn't exist")
-
-			return []byte(), err
-		}
-
-		content, err := ioutil.ReadFile(a.config.LDAP.Data)
-
-		if err != nil {
-			a.logger.Error().
-				Str("path", a.config.LDAP.Data).
-				Msg("Failed to read data file")
-
-			return []byte(), err
+// Open just implements the HTTP filesystem interface.
+func (a assets) Open(original string) (http.File, error) {
```

Comment:
Having this http file handler doesn't make sense at all for this extensions. Why have you replaced the Data function?